docs/examples/grpc/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang as build
+
+ADD https://raw.githubusercontent.com/grpc/grpc-go/master/examples/features/reflection/server/main.go /
+
+RUN apt update && \
+  apt upgrade -y && \
```

Comment:
It's unnecessary.

Reply:
removed apt update & apt upgrade

Reply:
No, just remove apt upgrade

Reply:
ok. will remove only upgrade. it will show up in next squashed push